src/polytope/github/repository/InternalCode.py

```diff
@@ -0,0 +1,26 @@
+from enum import Enum, auto
+
+
+class GithubRepositoryInternalCode(Enum):
+    """! Internal status code of Github CRUD requests."""
+
+    # Success.
+    Success = auto()
+    # create cannot modify repository name.
+    ForbiddenNameChangeOnCreation = auto()
+    # config validation failed.
```

Comment:
(suggestion) No need to do so, but please match the capitalization of the first word in each comment. (`Success` -> `success`)